drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -9,6 +10,7 @@
 #include <drivers/clock_control.h>
 #include <sys/util.h>
 #include <drivers/clock_control/stm32_clock_control.h>
+#include "stm32h7xx_ll_utils.h"
```

Comment:
I missed this one.
These inclusions should be done in `soc/arm/st_stm32/stm32h7/soc.h` under appropriated #ifdef.
And actually, it's already there, so you just need to add the right flag.

Reply:
Sounds like removing the line doesn't create any inclusion warning nore undefined function references. So this is to be removed.

Reply:
@erwango Another idea is just to include stm32h7xx_ll_utils.h if CONFIG_CLOCK_CONTROL is set

Reply:
Done, you can check if you like this one.